src/core/os/lnx/dri3/dri3Loader.h

```diff
@@ -1062,11 +1062,15 @@ class Dri3Loader
 
     void   SpecializedInit(Platform* pPlatform);
 
+    xcb_extension_t* GetXcbDri2Id() const;
+
     xcb_extension_t* GetXcbDri3Id() const;
 
     xcb_extension_t* GetXcbPresentId() const;
 
 private:
+    xcb_extension_t* m_pXcbDri2Id;
```

Comment:
It should be declared right after m_pXcbPresentId or there is a warning because of the mismatch between declaration order and initialization order. 
Is it generated by dri3Loader.py? If yes, there is probably a bug with dri3Loader.py. I tried dri3Loader.py with your proc file at my end, and m_pXcbDri2Id is following m_pXcbPresentId.

Reply:
I have made a second commit added to pull request to correct the indentation